src/WCSimAmBeGen.cc

```diff
@@ -0,0 +1,135 @@
+#include "WCSimAmBeGen.hh"
+#include "WCSimPrimaryGeneratorMessenger.hh"
+#include "WCSimPrimaryGeneratorAction.hh"
+
+#include "G4ParticleGun.hh"
+#include "G4PhysicalConstants.hh"
+#include "G4RandomDirection.hh"
+#include "G4SystemOfUnits.hh"
+#include "G4SPSEneDistribution.hh"
+#include "G4SPSRandomGenerator.hh"
+#include "G4ParticleTable.hh"
+#include "G4Neutron.hh"
+#include "G4Gamma.hh"
+
+#include <random>
+
+using namespace std;
+
+G4double WCSimAmBeGen::gammaProbabilities[3] = {0.26, 0.65, 0.08};
+G4double WCSimAmBeGen::gammaEnergies[3] = {0.0, 4.4, 7.7};
```

Comment:
```suggestion
G4int WCSimAmBeGen::nGammaOutcomes = 3;
G4double 
WCSimAmBeGen::gammaProbabilities[nGammaOutcomes] = {0.26, 0.65, 0.08};
G4double WCSimAmBeGen::gammaEnergies[nGammaOutcomes] = {0.0, 4.4, 7.7};
```
Do use a better name for `nGammaOutcomes` if you have one!